### PR TITLE
[WIP] Add draft js playground

### PR DESCRIPTION
**Pull request by juliankrispel**
**Summary**

Must haves:
- [x] add an playground to `examples`
- [x] integrate draft js editor
- [x] Button to update draft js editor from codemirror content
- [x] Button to import draft js contentState into codemirror
- [x] display current contentState
- [ ] integrate basic rich text editing into draft editor
- [ ] integrate codemirror for html and json editing
- [ ] use url query parameters to toggle feature flags

Nice to haves:
- [x] resizable panels
- [ ] use visualisation tool for displaying any json structures (like contentState)
- [ ] add option for viewing other parts of editorState other than contentState, like selection, undoStack, etc
- [ ] style nicely
- [ ] use localstorage to remember last configuration?

### CODE REVIEW

**Comment by flarnie** [comment]
I think if we get CI passing then this could be merged without adding all the remaining unchecked items. The demo you shared was looking good! :)
I also wonder if the code for displaying the editorState on the right could be useful as a stand-alone library, for displaying any Immutable/JSON data. Just a thought.

File: examples/draft-0-10-0/playground/src/DraftJsPlaygroundContainer.react.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
+**
```
Comment (mitermayer) [inline examples/draft-0-10-0/playground/src/DraftJsPlaygroundContainer.react.js]: @juliankrispel  this is missing a '/' on the opening comment :)

**Comment by mitermayer** [comment]
I tryed to update this PR myself but was not allowed to push to your repo.

**Comment by mitermayer** [comment]
i will import this PR and fix it manually inside phabricator

**Pull request closed without merging**